packaging/nuget/NServiceBus.Unity.nuspec

```diff
@@ -16,7 +16,7 @@
     <tags>$tags$</tags>
     <dependencies>
       <dependency id="NServiceBus" version="[6.0.0, 7.0.0)" />>
-      <dependency id="Unity" version="[5.8.6, 6.0.0)" />
+      <dependency id="Unity" version="[5.8.6, 5.9.0)" />
```

Comment:
Just an FYI, but this won't prevent someone from trying to use the prerelease versions of 5.9.0 that are nuget.org.

Reply:
is that really a scenario we need to prevent? are you suggesting to use 5.9.0- instead?

Reply:
I don't think there's even a valid workaround to prevent NuGet's insane behavior right now, because the proposed workaounds on the issue I opened about it don't seem to work, or if they do, won't be accepted by nuget.org.

I was just pointing out that this won't fully protect us from someone trying to use a 5.9.0 release.